### PR TITLE
CLOUDP-247419 Stop publishing to the server 4.4 repo

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Stop publishing to the server 4.4 repo

_Jira ticket:_ CLOUDP-247419

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: build/ci/release.yml [hunk 1]
```diff
@@ -189,14 +189,13 @@ functions:
           - revision
           - created_at
           - ext
-          - signing_auth_token_44
-          - signing_auth_token_46
```
Comment (gssbzn) [inline build/ci/release.yml:193]: I think we use this for the QA repo

Comment (fmenezes) [reply build/ci/release.yml:193]: what do you mean by QA repo? do you see any pointers in the code?

Comment (gssbzn) [reply build/ci/release.yml:193]: for snapshots it goes to a testing repo like https://repo.mongodb.com/apt/ubuntu/dists/jammy/mongodb-enterprise/development/ the logic used to be by the key being anything but a stable version if I remember correctly

Comment (fmenezes) [reply build/ci/release.yml:193]: I've added it back but so far I can't find a pointer to 4.6

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**